lib/install/is-fs-access-available.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+var fs = require('fs')
+var process = require('process')
+var semver = require('semver')
+
+var isWindows = process.platform === 'win32'
+
+// fs.access first introduced in node 0.12 / io.js
+if (!fs.access) {
+  module.exports = false
+} else if (!isWindows) {
+  // fs.access always works on non-Windows OSes
+  module.exports = true
+} else {
+  // The Windows implementation of `fs.access` has a bug where it will
+  // sometimes return access errors all the time for directories, even
```

Comment:
does it matter that the fix in io. js was to always return true in windows for directories? 
Are you just relying on someone adding functionality to fs.access? what happens if it doesnt detect access..  will npm not roll back correctly?